torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -419,6 +419,32 @@ def pad_bounding_box(
 crop_image_pil = _FP.crop
 
 
+def crop_bounding_box(
+    bounding_box: torch.Tensor,
+    format: features.BoundingBoxFormat,
+    image_size: Tuple[int, int],
+    top: int,
+    left: int,
+    height: int,
+    width: int,
+) -> torch.Tensor:
+    pass
+
+    shape = bounding_box.shape
+
+    bounding_box = convert_bounding_box_format(
+        bounding_box, old_format=format, new_format=features.BoundingBoxFormat.XYXY
+    ).view(-1, 4)
+
+    # Crop and optionally pad:
```

Comment:
What you mean optionally pad?

Reply:
Crop transform on images does explicitly padding if params suggest that. In case of cropping bboxes padding is implicit and we do not need to call `pad_bounding_box` for that. For example, if left is negative (=padding) then it will be implicitly done with the same operation as for cropping. Hope to be a bit more clear.